src/cli/Cli.ts

```diff
@@ -175,7 +190,7 @@ export class Cli {
       await this.commandToExecute.command.processOptions(optionsWithoutShorts.options);
     }
     catch (e: any) {
-      return this.closeWithError(e.message, optionsWithoutShorts, false);
+      this.closeWithError(e.message, optionsWithoutShorts, false);
```

Comment:
Is there any reason to remove this `return` clause? If there was an error processing command's options, then we should not proceed with command's execution and close the CLI with an error. We do this inside the `closeWithError` method, but the `return` here makes this clear that the rest of the code won't execute.